k8s/gateway/vald/ing.yaml

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
NOTE:
Because the apiVersion is updated, the `servicePort` field should also be updated.
But it is okay to ignore it because this file will be automatically updated by CI when a new release is created.

Reply:
I know. we can ignore it this time.